modules/opensimplex/doc_classes/NoiseTexture.xml

```diff
@@ -31,6 +31,9 @@
 		<member name="noise" type="OpenSimplexNoise" setter="set_noise" getter="get_noise">
 			The [OpenSimplexNoise] instance used to generate the noise.
 		</member>
+		<member name="noise_offset" type="Vector2" setter="set_noise_offset" getter="get_noise_offset" default="Vector2( 0, 0 )">
+			An offset used to specify the noise space coordinate of the top left corner of the generated noise. This value is ignored if [code]seamless[/code] is enabled.
```

Comment:
I didn't notice this at first, but you should use `[member]` references so they're clickable:

```suggestion
			An offset used to specify the noise space coordinate of the top left corner of the generated noise. [member noise_offset] is ignored if [member seamless] is [code]true[/code].
```

Reply:
Thanks for the heads up. I've updated this. I'll be sure to take a look at the documentation standards for future PRs.